src/wrappers.ts

```diff
@@ -0,0 +1,231 @@
+// Copyright 2021 Google LLC
```

Comment:
Could we rename `wrappers.ts` to something more descriptive like `functionWrappers.ts`?

Reply:
sure